src/java/org/apache/cassandra/repair/state/SessionState.java

```diff
@@ -42,14 +54,18 @@ public enum State
     public final String keyspace;
     public final String[] cfnames;
     public final CommonRange commonRange;
+    private final CoordinatorState coordinator;
     private final ConcurrentMap<UUID, JobState> jobs = new ConcurrentHashMap<>();
 
     public final Phase phase = new Phase();
 
-    public SessionState(Clock clock, TimeUUID parentRepairSession, String keyspace, String[] cfnames, CommonRange commonRange)
+    private static final long EMPTY_SIZE = ObjectSizes.measure(new SessionState(new CoordinatorState(SharedContext.Global.instance.clock(), 0, "", new RepairOption(RepairParallelism.SEQUENTIAL, false, false, false, 8, Collections.emptySet(), false, false, false, PreviewKind.NONE, false, false, false, false)), SharedContext.Global.instance.clock(), "", new String[] {}, new CommonRange(ImmutableSet.of(FBUtilities.getBroadcastAddressAndPort()), Collections.emptySet(), Range.rangeSet(new Range<>(new Murmur3Partitioner.LongToken(Long.MIN_VALUE), new Murmur3Partitioner.LongToken(Long.MIN_VALUE))))));
```

Comment:
since you do `ObjectSize.measure` it would be good to do `null` than define so much that isn't actually needed...